pom.xml

```diff
@@ -21,6 +21,13 @@
         <sonar.dynamic>reuseReports</sonar.dynamic>
```

Comment:
CNR entry?

Reply:
Fixed in b4d36b67676ff98755b268d2fecb44f3888a75ea